packages/infrastructure/variables.tf

```diff
@@ -0,0 +1,44 @@
+variable "location" {
+  type        = string
+  description = "Primary Location"
+  default     = "westeurope"
+}
+
+variable "name" {
+  type        = string
+  description = "name"
+  default     = "alvtime"
+}
+
+variable "env" {
+  type        = string
+  description = "environment"
+}
+
+variable "aks_service_principal_client_id" {
+  type    = string
+  description = "Azure Active Directory App registration (service principal) id"
+}
+
+variable "aks_service_principal_client_secret" {
+  type      = string
+  description = "Azure Active Directory App registration (service principal) client secret"
```

Comment:
Could we either store this in a Azure key vault, or somehow fetch this using a Terraform data source based on the  `aks_service_principal_client_id`?

Reply:
key vault

Reply:
Done